context/app/static/js/components/Search/Search.jsx

```diff
@@ -1,21 +1,15 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
-import { ExistsQuery, BoolMustNot } from 'searchkit';
 import SearchWrapper from './SearchWrapper';
 import { readCookie } from '../../helpers/functions';
 import './Search.scss';
-import { donorConfig, sampleConfig, datasetConfig } from './config';
+import { donorConfig, sampleConfig, datasetConfig, lastModifiedSort } from './config';
 // eslint-disable-next-line import/named
-import { filter, checkboxFilter } from './utils';
+import { filter } from './utils';
 import AncestorNote from './AncestorNote';
 import LookupEntity from '../../helpers/LookupEntity';
```

Comment:
Perhaps for another PR, but could we make this a hook? We can also change the import path to `helpers/LookupEntity`, the import statement will have to be moved below the import of readCookie due eslint import order issues.

Reply:
Filed #860, and I'll change the path here.